compact/ecal/backward_PbWO4.xml

```diff
@@ -39,11 +49,135 @@
             r12max="EcalEndcapN_structure_ring_max"
             SFlength="EcalEndcapN_structure_ring_length"
             CMlength="EcalEndcapN_CrystalModule_length + 2.*EcalEndcapN_CrystalModule_wrap_thickness"
+            NEEMC_PR="EcalEndcapN_Protate"
```

Comment:
Stick to one variable name scheme. You have some that are all caps NEEEMC_PR, some that are lowercase underscore. Don't be redundant in fields: that this is the NEEMC is clear from the plugin so just focus on the parameters inside the NEEMC.

Reply:
modified the variable names